hooks/secondary_pre_publish_tk-3dsmaxplus.py

```diff
@@ -82,16 +82,16 @@ def execute(self, tasks, work_template, progress_cb, user_data, **kwargs):
             item = task["item"]
             output = task["output"]
             errors = []
+            app = self.parent
 
             # report progress:
             progress_cb(0, "Validating", task)
 
-            # pre-publish item here, e.g.
-            #if output["name"] == "foo":
-            #    ...
-            #else:
-            # don't know how to publish this output types!
-            errors.append("Don't know how to publish this item!")
+            if output["name"] == "alembic_cache":
```

Comment:
It's confusing, but the output name ties to the scene_item_type in the secondary publish item definition in the config. In this case, the output name is "alembic_cache" and the scene_item_type is "geometry". So we get an output item of name "alembic_cache" for each "geometry" scene item that was reported.